docs/test_cases/TestSuite24.md

```diff
@@ -70,6 +70,15 @@ The end user can see the details of a reused BIE node that he does not own, and
 #### Test Assertion #24.1.13
 The end user can express a reusing BIE that reuses a BIE in WIP state and owned by a different user.
 
+#### Test Assertion #24.1.14
+The end user won't remove a higher level reference to a reused BIE when removing a different reused BIE at lower level. 
+
+#### Test Assertion #24.1.15
+Enable reusable global schema when a BIE references a reusable BIE within a nested reused BIE. The reused BIE can be expressed as its own global property and referenced within the nested BIE.
+
+#### Test Assertion #24.1.16
+Retain enabled properties, including their primitives, cardinality, and context defintions of a reused BIE when user disables its reuse. 
```

Comment:
Retain all enabled properties under the reused BIE hierarchy when the user clicks the 'Retain Reused BIE' context menu.